app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt

```diff
@@ -13,12 +16,16 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
+
 @RunWith(AndroidJUnit4::class)
 class SearchTETest : BaseTest() {
 
     @get:Rule
     val rule = ActivityTestRule(SearchTEActivity::class.java, false, false)
 
+    private var mapboxMap: MapboxMap? = null
+    lateinit var mapboxIdlingResourceRule: MapIdlingResourceRule
```

Comment:
shouldn't this watcher have the annotation @get :Rule?

Reply:
If I put the @get:Rule annotation and instantiate in here, the MapIdlingResource does not finds the mapView because the activityTestRule is null. Meanwhile instantiating the MapIdlingResourceRule in the test method does not have this problem.

@JaimeToca and I will take a look at this, to make it right.

Reply:
@ferdyrod If you're not using get:Rule. Then I think you can remove the rule and just init the idling in setup

Reply:
In the end that is what I ended up doing.